src/geometry.cpp

```diff
@@ -336,10 +336,11 @@ void cross_lattice(Particle& p, const BoundaryInfo& boundary)
     // The particle is outside the lattice.  Search for it from the base coords.
     p.n_coord() = 1;
     bool found = exhaustive_find_cell(p);
-    if (!found && p.alive()) {
-      p.mark_as_lost(fmt::format("Could not locate particle {} after "
-                                 "crossing a lattice boundary",
-        p.id()));
+
+    if (!found) {
```

Comment:
Is there a case where the particle might not be alive but this function is called nonetheless? It seems like even if so, it doesn't hurt to mark it as lost again, but wanted to make sure this was intentional that the `&& p.alive()` was removed.

Reply:
To answer my own question, I think this is because the method now uses a Geometron rather than a Particle, so there is no concept of aliveness. I don't think it makes a difference then, provided the Particle's `mark_as_lost()` function is getting called.

Reply:
Yes, you're exactly right! I was thinking that geometry specs should not care about aliveness, giving a bit more separation of responsibilities between physics and geometry code.